source/standards/source-code/use-github.html.md.erb

```diff
@@ -52,12 +52,16 @@ Consider protecting the `.github/workflows` folder by using [a CODEOWNERS file](
 Consider creating a Workflow Template in the [alphagov workflow folder](https://github.com/alphagov/.github/tree/main/workflow-templates) if you need to share a similar workflow between many repositories.
 
 [Create your own local actions](https://docs.github.com/en/actions/creating-actions/about-actions) wherever possible. If using GitHub-owned actions, [pin to a specific version](https://docs.github.com/en/actions/learn-github-actions/finding-and-customizing-actions#using-release-management-for-your-custom-actions).
+Pinned versions should include the semver version in a comment next to the SHA, helping humans understand which versions we are pinned to.
+Where possible, allow automated dependency management tools to scan these version comments and suggest updates.
```

Comment:
```suggestion
- You have pinned the specific version in your workflow and in the repository settings, using a Git commit SHA, and include the version tag as a comment.
- You have dependabot configured to ensure that any security or functional updates to those actions are kept up to date (e.g. deprecation of language version) 
```

Reply:
These are @andyloughran's changes from https://github.com/alphagov/gds-way/pull/874/files

Posting here so we can discuss what's worth pulling in?

Reply:
Aside from the specific mention of dependabot, I think we're saying the same thing here.

Open to others to weigh in on which reads better. I'm not notorious for my clear writing style.

Reply:
I think I prefer yours Huw; in the cold light of day 'repository settings' doesn't mean much..